plugins/other/pacman

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+# DESCRIPTION
```

Comment:
Please use the POD style documentation header. Maybe take a look at `plugins/munin/munin_events`.
This will allow the usage of `munindoc pacman`.